slackbridge/messages.py

```diff
@@ -69,11 +69,16 @@ def resolve(self):
                             self.raw_message['text'],
                         )
                     if subtype == 'file_share':
-                        return self._post_to_fluffy(
+                        file = self.raw_message['file']
```

Comment:
The syntax highlighter seems to think `file` is a reserved name here, but I think that should be fine? Does look a bit strange though.

Reply:
`file` was an alias for `open` in Python 2.